site/app/templates/grading/electronic/NavigationBar.twig

```diff
@@ -5,19 +5,23 @@
 
         {# If the student is in our section, add a clickable previous arrow, else add a grayed out one. #}
         {% if studentNotInSection %}
-            <i title="Go to the previous student" class="fas fa-chevron-left icon-header" style="color:grey"></i>
+            <i title="Previous ungraded student" class="fas fa-backward icon-header" style="color:grey"></i>
+            <i title="Previous student" class="fas fa-caret-left icon-header" style="color:grey"></i>
         {% else %}
-            <a href="javascript:void(0);" onclick="gotoPrevStudent();" data-href="{{ prev_student_url }}" id="prev-student"><i title="Go to the previous student" class="fas fa-chevron-left icon-header"></i></a>
+            <a href="javascript:void(0);" onclick="gotoPrevStudent(true);" data-href="{{ prev_ungraded_student_url }}" id="prev-ungraded-student"><i title="Previous ungraded student" class="fas fa-backward icon-header"></i></a>
+            <a href="javascript:void(0);" onclick="gotoPrevStudent();" data-href="{{ prev_student_url }}" id="prev-student"><i title="Previous student" class="fas fa-caret-left icon-header"></i></a>
         {% endif %}
 
         {# Back to list button #}
         <a href="{{ home_url }}"><i title="Go to the main page" class="fas fa-home icon-header" ></i></a>
 
         {# If the student is in our section, add a clickable next arrow, else add a grayed out one. #}
         {% if studentNotInSection %}
-            <i title="Go to the next student" class="fas fa-chevron-right icon-header" style="color:grey"></i>
+            <i title="Next student" class="fas fa-caret-right icon-header" style="color:grey"></i>
+            <i title="Next ungraded student" class="fas fa-forward icon-header" style="color:grey"></i>
         {% else %}
-            <a href="javascript:void(0);" onclick="gotoNextStudent();" data-href="{{ next_student_url }}" id="next-student"><i title="Go to the next student" class="fas fa-chevron-right icon-header"></i></a>
+            <a href="javascript:void(0);" onclick="gotoNextStudent();" data-href="{{ next_student_url }}" id="next-student"><i title="Next student" class="fas fa-caret-right icon-header"></i></a>
+            <a href="javascript:void(0);" onclick="gotoNextStudent(true);" data-href="{{ next_ungraded_student_url }}" id="next-ungraded-student"><i title="Next ungraded student" class="fas fa-forward icon-header"></i></a>
```

Comment:
I know this wasn't part of your issue, but you should change the void(0) to the functions you call with onclick so that keyboards can use it. e.g. `href="javascript:gotoNextStudent();"` instead of `href="javascript:void(0);" onclick="gotoNextStudent();"`

Reply:
I made the javascript changes you requested.  I didn't add any toast messages because I don't think I'll have time to get them working correctly in all cases.  For example when there are no more students left to grade going forward, but some still exist in the beginning of the grading order.